src/utils.ts

```diff
@@ -24,7 +24,11 @@ export const isV2Document = (document: any): document is v2.OpenAttestationDocum
 };
 
 export const isV3Document = (document: any): document is v3.OpenAttestationDocument => {
-  return !!document["@context"];
+  return !!document["@context"] && !!document.openAttestationMetadata;
+};
+
+export const isV4Document = (document: any): document is v4.OpenAttestationDocument => {
+  return !!document["@context"] && !document.openAttestationMetadata;
 };
```

Comment:
Shall we switch to using the type guards provided by the `open-attestation` library?

https://github.com/Open-Attestation/open-attestation?tab=readme-ov-file#utils

Reply:
oh, I thought Lawrence found some issue and switched over from those utils in https://github.com/Open-Attestation/decentralized-renderer-react-components/pull/125?

Reply:
Yeah I think it'll only work if we polyfill inside `example/decentralized-renderer/webpack.config.js`:

```javascript
module.exports = {
  //...
  resolve: {
    extensions: [".js", ".jsx", ".ts", ".tsx"],
    
    /* Add the following fallback */
    fallback: {
      crypto: require.resolve("crypto-browserify"),
      stream: require.resolve("stream-browserify"),
    },
  },
}c
```

Reply:
Ok will remove my reference to OA utils